web-portfolio.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8"> 
+	<title>Eleanor Stevens</title>
+	<link href="portfolio-stylesheet.css" type="text/css" rel="stylesheet">
+	<link href="normalize.css" type="text/css" rel="stylesheet">  <!-- this means the style is universal across all browsers -->
+	<link href="DINNextLTPro-Regular.ttf" type="text/ttf" rel="stylesheet">
+	<link href="https://github.com/elstevens23" rel="stylesheet">
+	
+</head>
+<body>
+<section class="navBar">
+	<nav>
+		<a href="portfolio.html">HOME</a>
+		<a href="about-me.html">ABOUT ME</a>
+		<a href="web-portfolio.html">PORTFOLIO</a>
+		<a href="contact.html">CONTACT ></a>
+	</nav>
+	
+	<h1 class="pageTitle"> <a href="portfolio.html">Eleanor Stevens</a> </h1>
+	<!-- <h2 class="subheading">WEBSITE DESIGN & DEVELOPMENT</h2> -->
+	<div class="right">
+	</div>
+	
+</section>
+
+<section class="portfolio-page">
+	<div class="projects-header">
+		<h1>Projects</h1>
+	</div>
+	
+	<div class="gallery-container">
+		
+		<div class="gallery-item"><a href="#"><img src="cat-grid.jpg" width="400" height="250"><div class="text-wrapper"><h4 class="name">Cover Name</h4><h5 class="project-title">Cover Subheading</h5></div></a></div>
```

Comment:
you need to break this onto multiple lines, its difficult to read/maintain like this